docs/deprecated.md

```diff
@@ -5,15 +5,6 @@ status: deprecated
 
 These features are phased out due to better alternatives or potential issues in future versions. Deprecated functionalities are supported for **three subsequent releases**, providing time for users to transition to updated methods.
 
-- [`Detections.from_roboflow`](detection/core.md/#supervision.detection.core.Detections.from_roboflow) is deprecated and will be removed in `supervision-0.22.0`. Use [`Detections.from_inference`](detection/core.md/#supervision.detection.core.Detections.from_inference) instead.
```

Comment:
Wouldn't be nice keep old deprecations in case of If you need to check history (kinda easier to check what is change or removed)

Reply:
Good idea, let's start that tradition. Added a section for removals in this doc.

Reply:
Awesome thank you.